reflex/reflex.py

```diff
@@ -680,6 +680,33 @@ def deploy(
     )
 
 
+@cli.command()
+def demo(
+    frontend_port: str = typer.Option(
+        "3001", help="Specify a different frontend port."
+    ),
+    backend_port: str = typer.Option("8001", help="Specify a different backend port."),
+):
+    """Run the demo app."""
+    console.set_log_level(constants.LogLevel.ERROR)
+
+    # Show system info
+    exec.output_system_info()
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        try:
+            os.chdir(tmp_dir)
+            os.system(
```

Comment:
Use `processes.new_process` for instead of directly calling `os`.

Also - will this work in cases such as poetry where they haven't aliased their `reflex` command correctly?